### DISCUSSION TITLE
Venue Lexicon Kickoff

### DISCUSSION THREAD

**Discussion by bmann** [lexicon-community/lexicon#13] [Ideas]
We just wrapped our call on geo data on AT Protocol https://smokesignal.events/did:plc:lehcqqkwzcwvjvw66uthu5oq/3lcgd3vggz72n

One of the take aways was to get specific here and see who wants to collaborate on a shared Lexicon around geo. 

I know that [Nosh Delivery](https://www.noshdelivery.co/) was also looking at this.

One of the design goals would be to leverage the new [Foursquare Open Source Places](https://location.foursquare.com/resources/blog/products/foursquare-open-source-places-a-new-foundational-dataset-for-the-geospatial-community/). I also have friends at Mapbox and other organizations that might want to participate.

The output of this would be a high quality Lexicon definition and supported tooling so that ATproto apps can use a shared Venue layer, and thus also aggregate content around Venues.

My personal interest is around Venue (business or POI), and I'd like to experiment with a "food spotting" type of record that uses it. This means roughly:
* name
* address (which is short for how to get there)
* notes on locating / getting in (e.g. buzz 301 and come to the third floor, in the food court next to the escalator)

Open questions for me:
* external identifier (eg foreign key to 4SQ places identifier)
* include lat/long or derive it?

This becomes part of a record -- e.g. a Smoke Signal Event or a Food Spotting Spot -- anything around a business claiming a record and having an associated profile with extra information I think would be a separate record / profile owned by that business.

Please leave a comment if this is interesting, especially with what your use case would be for a Venue lexicon.

**Comment by steveklabnik** [comment]
With the announcement of Foursquare shutting down (but not Swarm), I've been thinking heavily about wanting to mirror my Swarm data into my PDS.

However, this would end up being *private* data, not public, so would likely want to wait until that's solved first. But still interested in thinking about what would go in there.

I don't have any particular domain expertise in geo stuff, though, so I'm happy to participate but don't know how much insight I'll have.

**Reply by bmann** [reply]
Yeah, I'm focusing on public Venue info for this first Lexicon.

**Comment by bmann** [comment]
Just looking at Schema.org https://schema.org/docs/schemas.html

* Place https://schema.org/Place
* LocalBusiness https://schema.org/LocalBusiness
* Restaurant https://schema.org/Restaurant
